db/seeds/source_types.yml

```diff
@@ -123,6 +123,34 @@ ansible-tower:
         :condition:
           :when: endpoint.verify_ssl
           :is: true
+google:
+  :product_name: Google Cloud Engine
+  :vendor: Google
+  :icon_url: "/apps/chrome/assets/images/partners-icons/google-cloud-engine.svg" # TODO
+  :schema:
+    :authentication:
+    - :type: username_password
```

Comment:
I think we should describe it more accurately as we don't actually provide any username or password - `project_id` maybe ?.

Reply:
changed to `project_id_service_account_json`